android/src/main/java/com/swmansion/rnscreens/gamma/tabs/TabScreenViewManager.kt

```diff
@@ -172,6 +168,31 @@ class TabScreenViewManager :
         value: Boolean
     ) = Unit
 
+    // Android specific
+    @ReactProp(name = "tabBarItemBadgeVisible")
+    override fun setTabBarItemBadgeVisible(
+        view: TabScreen,
+        value: Boolean,
+    ) {
+        view.tabBarItemBadgeVisible = value
+    }
+
+    @ReactProp(name = "tabBarItemBadgeTextColor")
```

Comment:
All the color prop need the `customType = "Color" annotation argument, otherwise it wong work on old architecture. 

```suggestion
    @ReactProp(name = "tabBarItemBadgeTextColor", customType = "Color")
```